pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -237,3 +251,124 @@ func NewDomain(dom cli.VirDomain) (*api.Domain, error) {
 	domain.GetObjectMeta().SetUID(domain.Spec.Metadata.KubeVirt.UID)
 	return domain, nil
 }
+
+// ForkAndMonitor itself to give qemu an extra grace period to properly terminate
+// in case of virt-launcher crashes
+func ForkAndMonitor(commandNamePrefix string) error {
```

Comment:
is there a reason this needs to be in libvirt_helper.go? This is kind of specific to virt-launcher and not exactly libvirt.

Reply:
No. Will move it.